tools/walrus/README.md

```diff
@@ -49,3 +49,38 @@ The blob upload failed.
     - Timeout errors
 
 ---
+
+# `xyz.taluslabs.walrus.utils.verify_blob@1`
```

Comment:
please use `verify-blob` instead of `verify_blob`, this should be standardized everywhere.

Reply:
also the `.utils` namespace is meaningless I think, I'd rather go for `xyz.taluslabs.storage.walrus.x`